app/services/docker/containerService.js

```diff
@@ -18,10 +18,9 @@ angular.module('portainer.services')
     return deferred.promise;
   };
 
-  service.containers = function(all, filters) {
```

Comment:
I prefer to have see the signature of the `ContainerService.containers()` function as `ContainerService.containers(all, filters)` as the `all` parameter is actually separated from `filters` in the ContainerList operation: https://docs.docker.com/engine/api/v1.30/#operation/ContainerList